pwiz/utility/misc/Stream.hpp

```diff
@@ -57,7 +57,9 @@ using std::streampos;
 using std::streamoff;
 using std::streamsize;
 
+/* this breaks VS2019 usage

```

Comment:
We should revert this change before merging to master.

Reply:
The line of code is now uncommented out, I left the comment on breaking VS2019 in case it comes up again.